Update expectations from hack-a-thon feedback (#308)
cnerg/cnerg.github.com: community/manual/expectations.md

You have a right to the following from me as your advisor:
* Maintenance/support of your research tools and environment to maximize your
  research progress,

> * Provide financial support to attend conferences to present CNERG research

Reviewer: ```suggestion
* Provide financial support to attend conferences to present CNERG research,
```

> * Provide financial support to attend conferences to present CNERG research

Reviewer 2: ```suggestion
* Financial support to attend conferences to present CNERG research,
```
Parallelism: "Maintenance, Financial Aid, Timely Review", vs "Maintenance, Provide Financial Aid, Timely Review"

* Timely review of your research products,
* Access to my time to individually address questions and obstacles to your research,
* Constructive assessment of the quality and progress of your research,

> * Provide guidance for academic progress and plan, 

Reviewer 2: ```suggestion
* Guidance for your academic progress and plan, 
```
Parallelism again

under the expectations described here.  This will also provide you a time to
provide feedback to me on my role as an advisor. You will be asked to provide a
formal report of the following: Your progress during the previous time period,

> including a self-assessment of that work, Your goals for the coming semester, Any

Reviewer: ```suggestion
including a self-assessment of that work, your goals for the coming semester, any
```

Presenting your work at conferences is an important way to draw attention to
your contributions and leads to valuable interactions with peers about your
research.  We will work together to identify the most relevant conferences for

> your work and track the rhythm and deadlines to plan your participation. Both

Reviewer 2: I _think_ I understand what "the rhythm" is referring to, but it feels pretty unclear. I don't have a great suggestion on how to fix this, but I wanted to highlight it for potential rewording. (Also parallelism)

oral and poster presentations allow you to attach a personal element to your
work and convince the audience to learn more about your research through your
written publications. The quality of the presentations can be important in
establishing your professional identity.  We will work together to identify the
most important information to include and the most effective way to present it,

> both in oral and poster presentations. Over time you will become comfortable at

Reviewer 2: either "in both oral and poster presentations" or both in oral and in poster presentations"

these comments are a bit pedantic, feel free to ignore.

proficient at finding good quality articles that are relevant to your work.
Reading other people’s published work will also contribute to improved writing
skills.  A goal of a detailed reading of a few publications per month is a good
minimum standard.

### Seminars

The Nuclear Engineering & Engineering Physics department conducts a regular
seminar series with a wide range of topics from all the research areas of the
department.  These seminars are generally scheduled weekly and you should
receive notification from the department office.  All graduate students in the
NEEP department are expected to attend these seminars.  While it may be tempting
to dismiss some topics as unrelated to your work, this is an opportunity to
learn about a wide variety of interesting research.  More importantly, it is
common to find connections to your own work, even if they are weak connections,
and in so doing you will develop a deeper understanding of the work you are
pursuing.

### Internships

Students are welcome to pursue internships at companies and national
laboratories during their graduate studies, but these should be discussed and

> coordinated with their advisor. Important considerations include academic

Reviewer 2: ```suggestion
coordinated with me. Important considerations include academic
```
Since this is a CNERG document, "their advisor" should always be @gonuke , right?

milestones, student funding duration, and research deliverables.  As students
progress in their research, there will be a strong incentive for internships
to be aligned to their own research efforts to allow them to continue making
progress during the internship. In some instances, students have identified
their PhD research projects during an internship when early in their plan.

### Technical Conferences and Professional Networks

You are encouraged to attend technical conferences to as a way to engage with

> your technical community. We will discuss at which conferences make sense for you

Reviewer 2: ```suggestion
your technical community. We will discuss at which conferences it makes sense for you
```

The classic "wrote the first half of the sentence, did something else, and then finished the sentence in a slightly different way than you started it" issue.

Author: Actually - this was a classic "accept a suggestion from someone without reading how it affected the whole sentence" :)

to present your CNERG research, with funding from the grants that support that
research. You may also identify conferences that are valuable and interesting,
beyond those for which you will receive financial support. (The Graduate School
has [programs to request support for travel to
conferences.](https://grad.wisc.edu/funding/grants-competition/)) The personal

> interactions that are possible in face-to-face meetings are uniquely valuable

Reviewer 2: ```suggestion
interactions that are possible in face-to-face meetings at conferences are uniquely valuable
```
this one's maybe not the most important
